src/puppetlabs/puppetdb/cli/export.clj

```diff
@@ -10,8 +10,10 @@
   (:require [clj-http.client :as client]
             [clj-http.util :refer [url-encode]]
             [clj-time.core :refer [now]]
+            [clojure.java.io :as io]
```

Comment:
I suspect there might be a few of those lurking, I used the full name while prototyping and moving things around namespaces. I'll fix it up!